shared/common-adapters/text.desktop.js

```diff
@@ -133,9 +147,9 @@ export const styles = {
   },
   textBodySmall: {
     ...textCommon,
+    color: globalColorsDZ2.black40,
     fontSize: 14,
     lineHeight: '19px',
-    opacity: 0.4,
```

Comment:
@MarcoPolo  @cjb  the colors (black75/black40/etc) need to exist. they're used by other things so it should be shared. managing the opacity and the color separately i think might be error prone